internal/oci/runtime_oci.go

```diff
@@ -869,8 +869,6 @@ func (r *runtimeOCI) StopLoopForContainer(c *Container, bm kwait.BackoffManager)
 
 	startTime := time.Now()
 
-	go c.KillExecPIDs()
```

Comment:
@haircommander any reason this is a separate go routine?

Reply:
Not sure why. However, it probably shouldn't be.

Reply:
I didn't want to block the container stop, but if we do it after it should be fine